buildozer/targets/android.py

```diff
@@ -70,8 +69,8 @@ def __init__(self, *args, **kwargs):
             'app', 'android.arch', DEFAULT_ARCH)
         self._build_dir = join(
             self.buildozer.platform_dir, 'build-{}'.format(self._arch))
-        executable = sys.executable or 'python'
-        self._p4a_cmd = '{} -m pythonforandroid.toolchain '.format(executable)
+        _executable = executable or 'python'
```

Comment:
As a general comment, this is exactly why importing every name explicitly isn't necessarily the best idea - using up `executable` in the module namespace is fairly pointless, there's nothing wrong with referencing `sys.executable`. More generally, I think importing explicitly from `os`, `sys` etc. provides no benefit and if anything makes the code less clear.